client/a8c-for-agencies/sections/overview/body/hosting/pressable-offering.tsx

```diff
@@ -1,13 +1,13 @@
 import page from '@automattic/calypso-router';
 import { Button, FoldableCard, Gridicon } from '@automattic/components';
+import CoreBadge from '@automattic/components/src/core-badge';
```

Comment:
In places other than `client/dashboard`, is it recommended to import `CoreBadge` from `@automattic/components/src/core-badge` instead of `@automattic/components`?

Reply:
Not really sure. Is there a difference? They are used both in the code base, although the plain `@automattic/components` is used more. -cc @tyxla

Reply:
Yes! In that case, you can and should import it from `@automattic/components`.

Reply:
Good to know, thank you! I updated the imports.